pontoon/base/static/js/main.js

```diff
@@ -505,7 +512,9 @@ $(function() {
 
     listitems.sort(function(a, b) {
       return (val(index, a) < val(index, b)) ? -dir :
-        (val(index, a) > val(index, b)) ? dir : 0;
+        (val(index, a) > val(index, b)) ? dir :
+        (val(index, a, "unapproved") < val(index, b, "unapproved")) ? -dir :
```

Comment:
It feels weird to call `val` again with a single parameter different. Perhaps consider either splitting `val` up into multiple functions to avoid re-doing unnecessary work, or moving the code from `val` here instead of calling it.